user-operator/src/test/java/io/strimzi/operator/user/model/KafkaUserModelTest.java

```diff
@@ -226,9 +226,9 @@ public void testGenerateSecretGeneratesCertificateWhenNoSecretExistsProvidedByUs
         Secret generated = model.generateSecret();
 
         assertThat(new String(model.decodeFromSecret(generated, "ca.crt")), is("clients-ca-crt"));
-        assertThat(new String(model.decodeFromSecret(generated, "user.crt")), is("crt file"));
-        assertThat(new String(model.decodeFromSecret(generated, "user.key")), is("key file"));
-        assertThat(new String(model.decodeFromSecret(generated, "user.p12")), is("key store"));
+        assertThat(generated.getData().get("user.crt"), is(MockCertManager.entityCert()));
+        assertThat(generated.getData().get("user.key"), is(MockCertManager.entityKey()));
+        assertThat(generated.getData().get("user.p12"), is(MockCertManager.entityCertStore()));
```

Comment:
This nicely demonstrates the points I made on `MockCertManager`. You replaced something clearly mocked with something what is not valid anyway, but in a very sneaky way.